src/ResultView.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+
+const result_category = {
+  fontFamily: "Helvetica",
+  fontSize: 14,
+  color: "#7C7C72",
+};
```

Comment:
Instead of adding styles through `style={result_category}`, please assign the class name `sui-result__label` to the `span`s and place the following piece of CSS code in `App.css`:

```
.sui-result__label {
    color: "#7c7c72";
}
```

(Note that I removed font family and font size, since these two should be kept consistent. Let me know if this change results in monospace font.)